river/Control.zig

```diff
@@ -117,6 +117,7 @@ fn handleRequest(control: *zriver.ControlV1, request: zriver.ControlV1.Request,
                 };
                 defer if (err == command.Error.Other) util.gpa.free(failure_message);
                 callback.sendFailure(failure_message);
+                callback.destroy();
```

Comment:
Hmm, I think I'd rather have zig-wayland generate a `destroySendFailure()` that both sends the failure event and destroys the callback so that we can't forget to call destroy and potentially leak memory on the server side.